scripts/render.py

```diff
@@ -53,9 +59,11 @@ def _render_trajectory_video(
         rendered_resolution_scaling_factor: Scaling factor to apply to the camera image resolution.
         seconds: Length of output video.
         output_format: How to save output data.
+        camera_type: Camera projection format type.
+        render_width: Video width to render.
```

Comment:
Nit: doc order should match definition order

Reply:
Must have been one of the isort / black / pylint fixes, forgot to update docs